test/verify/machineslib.py

```diff
@@ -1345,6 +1345,20 @@ def installWithErrorTest(dialog):
                                              os_name=config.NOVELL_NETWARE_6,
                                              start_vm=False))
 
+            self.machine.execute("cwd=/var/lib/libvirt")
+            server = self.machine.spawn("python3 -m http.server 8080", "httpserver")
+
+            # Test detection of ISO file in URL
+            createTest(TestMachines.VmDialog(self, "subVmTestCreate20", sourceType='url',
```

Comment:
Please remove the VM name.

Reply:
Ah this change was not yet merged, https://github.com/cockpit-project/cockpit/pull/11491

Sorry :/
Leave it as is if this one will land first.